_posts/2023-05-24-v25-rc-testing.md

```diff
@@ -21,7 +21,19 @@ commit:
   essential that the rcs are thoroughly tested. This special review club
   meeting is for people who want to help with that vital review process.
 
+- This [Bitcoin Core Release Candidate Testing
+  Guide](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/25.0-Release-Candidate-Testing-Guide) provides guidance for testing the release candidate.
 
+  - It is recommended to go through the "Preparation" steps ahead of the meeting, especially if you want to compile from source.
+
+  - Verify and confirm the successful installation of v25rc2.
+
+  - The testing guide relies on the tools `jq` which are not installed by default on each platform. For example on macOS, you can install these ahead of time using `brew install`. Alternatively, you can also modify the instructions to avoid using these tools as they are not strictly necessary and/or can be replaced by other tools.
+
+  - For some of the tests, you might want to have the signet network fully synced beforehand, so that you will just copy the signet directory into  `/tmp/25-rc-test/` see [signet](https://en.bitcoin.it/wiki/Signet).
```

Comment:
```suggestion
  - For some of the tests, you might want to have the [signet](https://en.bitcoin.it/wiki/Signet) chain fully synced beforehand, so that you can just copy the signet directory into `/tmp/25-rc-test/` every time you run a test in a fresh environment.
```

Reply:
Sure, fixed.
Thanks